sui/transfer-object.js

```diff
@@ -38,17 +38,9 @@ async function processCommand(chain, options) {
     const tx = new Transaction();
     tx.transferObjects([`${objectId}`], tx.pure.address(recipient));
 
-    const result = await client.signAndExecuteTransaction({
-        transactionBlock: tx,
-        signer: keypair,
-        options: {
-            showObjectChanges: true,
-            showBalanceChanges: true,
-            showEvents: true,
-        },
-    });
+    const result = await broadcast(client, keypair, tx);
```

Comment:
Maybe let `broadcast` accept an optional message string so it can print the digest internally
```suggestion
    await broadcast(client, keypair, tx, 'Object Transferred');
```

Reply:
will do this in a separate pr, so it's cleaner to review.